rwgame/GameWindow.cpp

```diff
@@ -34,12 +36,20 @@ void GameWindow::create(const std::string& title, size_t w, size_t h, bool fulls
 		throw std::runtime_error("SDL_GL_CreateContext failed: " + sdlErrorStr);
 	}
 
+	if (mainwindow_ == nullptr) {
+		mainwindow_ = window;
+	}
```

Comment:
If you chose to do this, wouldn't it be easier to just keep track of the window which last grabbed the focus?